pkg/handler/custom_token.go

```diff
@@ -120,3 +120,35 @@ func (h *Handler) ListAllCustomToken(c *gin.Context) {
 
 	c.JSON(http.StatusOK, gin.H{"data": returnToken})
 }
+
+func (h *Handler) TokenCompare(c *gin.Context) {
```

Comment:
this is not`custom_token`, put in another handler
suggestion: group defi, coins -> `/defi/coins/compare`